src/app/components/user/UserWorkspaces/index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable relay/unused-fields */
```

Comment:
Please remove the `eslint-disable relay/unused-fields` directive or add a comment that explains why it is necessary

Reply:
Removed from this component, but it's still needed in the pagination file